### PR TITLE
added intro to README

### PR DESCRIPTION
**Pull request by mhowlett**
@gwenshap @edenhill @ewencp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
-Confluent.Kafka - Apache Kafka .NET client library
```
Comment (edenhill) [inline README.md]: I think we should keep this

File: README.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
-==================================================
+Confluent.Kafka
+===============
 
-**Confluent.Kafka** is a .NET client for [Apache Kafka](http://kafka.apache.org/) based on
-[librdkafka](https://github.com/edenhill/librdkafka).
+**Confluent.Kafka** is Confluent's .NET client for [Apache Kafka](http://kafka.apache.org/) and
```
Comment (edenhill) [inline README.md]: Confluent Inc's  ?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+[Confluent Platform](https://www.confluent.io/product/).
```
Comment (edenhill) [inline README.md]: the

File: README.md [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
 
-Derived from Andreas Heider's [rdkafka-dotnet](https://github.com/ah-/rdkafka-dotnet).
+Features:
+
+- **High performance** - Confluent.Kafka is a lightweight wrapper around
+[librdkafka](https://github.com/edenhill/librdkafka), a finely tuned C
+client.
+
+- **Reliability** - There are a lot of details to get right when writing a Kafka
```
Comment (edenhill) [inline README.md]: Apache Kafka

File: README.md [hunk 5]
```diff
@@ -1,10 +1,31 @@
...
+client. We get them right in a one place (librdkafka) and leverage this work
```
Comment (edenhill) [inline README.md]: "in a one place" sounds  weird, is that correct?

File: README.md [hunk 6]
```diff
@@ -1,10 +1,31 @@
...
+across all of our clients (also [python](https://github.com/confluentinc/confluent-kafka-python)
```
Comment (edenhill) [inline README.md]: spell out the full client name [confluent-kafka-python]..

File: README.md [hunk 7]
```diff
@@ -1,10 +1,31 @@
...
+and [golang](https://github.com/confluentinc/confluent-kafka-go)).
```
Comment (edenhill) [inline README.md]: dito

File: README.md [hunk 8]
```diff
@@ -1,10 +1,31 @@
...
+
+- **Supported** - Commercial support is offered by
+[Confluent](https://confluent.io/).
```
Comment (edenhill) [inline README.md]: Inc

File: README.md [hunk 9]
```diff
@@ -1,10 +1,31 @@
...
+
+- **Future proof** - Confluent, founded by the
```
Comment (edenhill) [inline README.md]: Inc

File: README.md [hunk 10]
```diff
@@ -1,10 +1,31 @@
...
+creators of Kafka, is building a [streaming platform](https://www.confluent.io/product/)
```
Comment (edenhill) [inline README.md]: Apache Kafka

File: README.md [hunk 11]
```diff
@@ -1,10 +1,31 @@
...
+with Kafka at it's core. It's high priority for us that client features keep
```
Comment (edenhill) [inline README.md]: Apache Kafka

its

File: README.md [hunk 12]
```diff
@@ -1,10 +1,31 @@
...
+pace with core Kafka and components of [Confluent Platform](https://www.confluent.io/product/).
```
Comment (edenhill) [inline README.md]: Apache Kafka

the Confluent plat..

File: README.md (hunk 2 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 3 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 4 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 5 above)
Comment (mhowlett) [inline README.md]: nope

File: README.md (hunk 6 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 7 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 8 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 9 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 10 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 11 above)
Comment (mhowlett) [inline README.md]: yep

File: README.md (hunk 12 above)
Comment (mhowlett) [inline README.md]: yep

**Comment by mhowlett** [comment]
didn't put in the inc's. changed title according to @gwenshap suggestion. also changed library name from Confluent.Kafka -> confluent-kafka-dotnet.

**Pull request merged**